AsyncDisplayKit/ASDisplayNode+Subclasses.h

```diff
@@ -65,6 +67,17 @@
  */
 @property (nonatomic, readonly, assign, getter=isInHierarchy) BOOL inHierarchy;
 
+/**
+ * @abstract Return the calculated layout.
+ *
+ * @discussion Ideal for use by subclasses in -layout, having already prompted their subnodes to calculate their size by
```

Comment:
Can you give a sample as our asdk and non asdk project have  lots of clips to bounds rounded corners

On Mon, Jul 6, 2015 at 8:40 AM, Huy Nguyen <notifications@github.com>
wrote:

>> @@ -65,6 +67,17 @@
>>   */
>>  @property (nonatomic, readonly, assign, getter=isInHierarchy) BOOL inHierarchy;
>>  
>> +/**
>> + * @abstract Return the calculated layout.
>> + *
>> + * @discussion Ideal for use by subclasses in -layout, having already prompted their subnodes to calculate their size by
> Similar to the modification block :) Its implementation can be extracted and should work outside of ASDK.
> On 6 Jul 2015 at 10:26:00 pm, markmark1 (notifications@github.com) wrote:
> In AsyncDisplayKit/ASDisplayNode+Subclasses.h:
>> @@ -65,6 +67,17 @@
>>   */
>>  @property (nonatomic, readonly, assign, getter=isInHierarchy) BOOL inHierarchy;
>>   
>> +/**
>> + * @abstract Return the calculated layout.
>> + *
>> + * @discussion Ideal for use by subclasses in -layout, having already prompted their subnodes to calculate their size by
> Thanks huy checking in general outside asdk how do you create rounded corners
> …
> —
> Reply to this email directly or view it on GitHub.
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/facebook/AsyncDisplayKit/pull/468/files#r33947874